deploy_requirements.txt

```diff
@@ -7,7 +7,7 @@ beautifulsoup4
 GoogleAppEngineCloudStorageClient
 
 # Cloud Endpoints Stuff
-oauth2client==3.0.0
+oauth2client==4.1.2
```

Comment:
We probably don't need to bump this all the way up - however, the service worker stuff I'm using to generate an OAuth token for dispatching notifications isn't available in that `3.0.0`